src/codegen/generators/expression_generator.rs

```diff
@@ -416,8 +421,8 @@ impl<'ink, 'b> ExpressionCodeGenerator<'ink, 'b> {
         // find the pou we're calling
         let pou = self.annotations.get_call_name(operator).zip(self.annotations.get_qualified_name(operator))
             .and_then(|(call_name, qualified_name)| self.index.find_pou(call_name)
-            // for some functions (builtins) the call name does not exist in the index, we try to call with the originally defined generic functions
-            .or_else(|| self.index.find_pou(qualified_name)))
+                // for some functions (builtins) the call name does not exist in the index, we try to call with the originally defined generic functions
```

Comment:
are these tabs deliberate here?

Reply:
I think it might make sense to run the precommit hook again on the entire base, but we can do it after this is merged in its own commit.

Reply:
Not intentional, this slipped in because rustfmt just gives up here